requirements.yml

```diff
@@ -0,0 +1,2 @@
+roles:
+  - name: linux-system-roles.firewall
```

Comment:
We really didn't want to do this: https://linux-system-roles.github.io/documentation/role-requirements.html
```
### Dependencies/Requirements

We can not use standard [role dependencies](https://docs.ansible.com/ansible/latest/playbook_guide/playbooks_reuse_roles.html#role-dependencies) as defined in meta/main.yml, because they do not lead only to installation of these dependencies, but also their invocation. But we do not know if we want to invoke them at the installation time nor what arguments we want to pass. Therefore we will use the collection requirements here.
```